guides/common/modules/con_redis-tuning.adoc

```diff
@@ -4,8 +4,12 @@
 = Redis tuning
 
 Redis is an in-memory data store.
-It is used by multiple services in {Project}.
-The Dynflow and Pulp tasking systems use it to track their tasks.
+ifdef::katello,orcharhino,satellite[]
+Foreman, the Dynflow tasking system, and Pulp use Redis for caching.
```

Comment:
Please add back that Dynflow uses Redis to track its tasks. I meant that **in addition** to that, Foreman and Pulp also use Redis for caching.